lib/ansible/modules/crypto/acme/acme_certificate.py

```diff
@@ -197,6 +197,15 @@
     type: bool
     default: no
     version_added: 2.6
+  retrieve_all_alternates:
+    description:
+      - "When set to C(yes), will retrieve all alternate chains offered by the ACME CA.
+         These will not be written to disk, be returned together with the main chain as
```

Comment:
"not be written to disk, be returned"
"not be written to disk, but be returned" or "not be written to disk. They will be returned"

Reply:
Good point!

Reply:
Changed it to "[...] to disk, but will be returned".